src/verifiers/issuerIdentity/didIdentityProof/didIdentityProof.test.ts

```diff
@@ -0,0 +1,33 @@
+import { OpenAttestationDidSignedDidIdentityProof } from "./didIdentityProof";
```

Comment:
it's not stated but it should work the same way for token registry ? and certificate store ?